src/modules/likeItems.js

```diff
@@ -0,0 +1,38 @@
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/ilWLjX7DiufqJ1ygUhcb/likes';
+const getnumberofLikes = async () => {
+  const cards = document.querySelectorAll('.cards');
+  const likeCount = document.querySelectorAll('.like-count');
+  await fetch(url)
+    .then((response) => response.json())
+    .then((json) => {
+      cards.forEach((card, index) => {
+        json.forEach((item) => {
+          if (item.item_id === card.id) {
+            likeCount[index].innerHTML = item.likes;
+          }
+        });
+      });
+    });
+};
+
+const interactLikeButton = () => {
+  const cards = document.querySelectorAll('.cards');
+  const likeBtns = document.querySelectorAll('.like-btn');
+  const likeCounts = document.querySelectorAll('.like-count');
+  likeBtns.forEach((btn, index) => {
+    btn.addEventListener('click', async () => {
+      const response = await fetch(url, {
+        method: 'POST',
+        body: JSON.stringify({ item_id: cards[index].id }),
+        headers: { 'Content-Type': 'application/json' },
+      });
+      if (response.ok) {
+        const json = await response.json();
+        likeCounts[index].textContent = json.likes;
+      }
+      await getnumberofLikes();
+    });
+  });
+};
```

Comment:
- [ ] Nice job making the required changes asked by the previous reviewer. Unfortunately, you have a new issue now.
Clicking the like button now does add a new like to the API server but the doesn't update likes on the screen until I restart the page.
As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=Involvement%20API%20and-,the%20screen%20is%20updated.,-When%20the%20user), Please fix that.

![chrome-capture-2023-2-26](https://user-images.githubusercontent.com/68796232/227747063-4c2c5eeb-4b33-487f-bf68-b4942414ebef.gif)